config.toml

```diff
@@ -181,16 +187,13 @@ project = "SUSE:SLE-15-SP4:Update"
 package = "python-requests"
 [origins.saltbundlepy-rpm-macros]
 project = "SUSE:SLE-15-SP4:Update"
-package = "python-macros"
-[origins.saltbundlepy-rpm-vercmp]
-project = "SUSE:SLE-15-SP4:Update"
-package = "python-vercmp"
```

Comment:
Can this be dropped? I still see it at https://build.opensuse.org/package/show/systemsmanagement:saltstack:bundle/saltbundlepy-rpm-vercmp

Reply:
It is used, but it is built from Github (as per https://github.com/openSUSE/salt/wiki/Salt-Bundle) - it has no upstream package in OBS